core/src/main/java/org/infinispan/conflict/MergePolicies.java

```diff
@@ -0,0 +1,20 @@
+package org.infinispan.conflict;
+
+import org.infinispan.container.entries.CacheEntry;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+public class MergePolicies {
+
+   public static final EntryMergePolicy PREFERRED_ALWAYS = (preferredEntry, otherEntries) -> preferredEntry;
+
+   public static final EntryMergePolicy PREFERRED_NON_NULL = (preferredEntry, otherEntries) -> {
+      if (preferredEntry != null || otherEntries.isEmpty()) return preferredEntry;
+
+      return (CacheEntry) otherEntries.get(0);
+   };
+
+   public static final EntryMergePolicy REMOVE_ALL = (preferredEntry, otherEntries) -> null;
```

Comment:
I don't have a specific use case in mind, but as the cost of implementation is low, I figured it was a worthwhile addition. It was also necessary to ensure that entries are removed from the cache if the merge policy returns null, so this policy (or a variation of it) would have to exist in the test code.